pkg/util/retry/retry.go

```diff
@@ -36,6 +36,7 @@ func UntilSuccessWithOptions(t test.TestHelper, options RetryOptions, f func(t t
 		if lastAttempt {
 			attemptHelper = t
 			f(attemptHelper)
+			//t.T().Fatalf(":(")
```

Comment:
I suppose you miss it here during tests :smiley:

Reply:
updated